mne/morph.py

```diff
@@ -468,7 +468,7 @@ def _check_dep(nibabel='2.1.0', dipy='0.10.1'):
                                                                         ver))
 
 
-def _morphed_stc_as_volume(morph, stc, mri_resolution=False, mri_space=True,
+def _morphed_stc_as_volume(morph, stc, mri_resolution, mri_space,
```

Comment:
Here the defaults never mattered because this is only used in one place and both params are passed